pkg/constants/constants.go

```diff
@@ -22,6 +22,9 @@ const (
 	// OktetoURLAnnotation indicates the okteto cluster public url
 	OktetoURLAnnotation = "dev.okteto.com/url"
 
+	// OktetoDevModeAnnotation indicates the development mode in use (sync or hybrid)
+	OktetoDevModeAnnotation = "dev.okteto.com/mode"
```

Comment:
@ifbyol Do we agree that this is the correct name for the annotation? 

The possible values are `sync` and `hybrid`

Reply:
Given that `dev.okteto.com` is the common prefix for our labels and annotations, `mode` doesn't specify too much about what it means or what it refers to. What about `dev.okteto.com/dev-mode`? 

Anyways, no strong opinion on the name of the annotation, so you can leave it as it is. The key part is being able to identify when an okteto up was sync or hybrid mode.

Reply:
👍  b0832a86ca453c1fced70b11f89487ce79ff7b18